Client/src/Components/Graph/GraphControls/Axes/AxesControl.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Button, Grid } from "@material-ui/core"
```

Comment:
Just a minor comment, Shouldn't the folder and file names be Axis instead of axes?

Reply:
And there is a code smell, don't forget to check it out

Reply:
Axes is the plural of axis since there's two of them.